npm/index.md

```diff
@@ -11,31 +11,22 @@ theme: sudodoki/reveal-cleaver-theme
 
 --
 
-# Where goes what?
+# What goes where?
 ## Node dependencies -> npm
 
 ## Front-end dependencies -> bower
 
 --
 
-##![NPM](npm.png)
---
-
 ## [NPM](https://npmjs.org/)
 
-+ Written by Isaac Z. Schlueter
-+ Total Packages: 49 435
-+ 2 667 474  downloads in the last day (circa 2.12.13)
-+ 30 663 295   downloads in the last week (circa 2.12.13)
++ Written by Isaac Z. Schlueter, Dennis Daniels
```

Comment:
I think you mistakenly thought this line refers to author of the slides, Isaas is author of npm.